Frameworks/UIKit/UITextField.mm

```diff
@@ -213,12 +221,69 @@ - (NSDictionary*)defaultTextAttributes {
 }
 
 /**
- @Status Caveat
- @Notes Fonts can be set, but will be ignored for rendering, instead default font will be used.
+ @Status Interoperable
 */
 - (void)setFont:(UIFont*)font {
     _font = font;
-    // TODO 7374333: need map UIFont with fontFamily/FontSize/FontWeight/FontStyle on target
+    [self _adjustFontSizeToFitWidthOrApply];
+}
+
+- (void)_applyFont:(UIFont*)font {
+    _passwordBox.fontFamily = _textBox.fontFamily = [WUXMFontFamily makeInstanceWithName:[font _compatibleFamilyName]];
+
+    // The following enums map from DWrite directly to Xaml
+    _passwordBox.fontStretch = _textBox.fontStretch = (WUTFontStretch)[font _fontStretch]; 
+    _passwordBox.fontStyle = _textBox.fontStyle = (WUTFontStyle)[font _fontStyle]; 
+    _passwordBox.fontSize = _textBox.fontSize = font.pointSize;
+
+    WUTFontWeight* weight = nil;
+
+    switch ([font _fontWeight]) {
```

Comment:
I was thinking for internal use, should this conversion ever need to be done elsewhere from UIFont to Xaml